packaging/leapp.spec

```diff
@@ -41,67 +51,75 @@ URL:        https://oamg.github.io/leapp/
 Source0:    https://github.com/oamg/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 BuildArch:  noarch
 
+Requires: %{leapp_python_name}-%{name} = %{version}-%{release}
+%{?python_disable_dependency_generator}
+
 %if !0%{?fedora}
-%if %{with python3}
-Requires: python3-%{name} = %{version}-%{release}
-%else
-Requires: python2-%{name} = %{version}-%{release}
-%endif
+# TODO: We plan to drop the dependency once we get ack for the planned changes.
+# # In such a case, leapp will not install leapp-repository automatically. For
+# # the full installation, people will have to install a metapackage which will
+# # be part of a particular leapp-repository (e.g. leapp-upgrade metapackage)
+# # Currently the leapp utility does not provide any subcommands (but help)
+# # so it doesn't require the leapp-repository anymore.
 
 # Just ensure the leapp repository will be installed as well. Compatibility
 # should be specified by the leapp-repository itself
 Requires: leapp-repository
 %endif # !fedora
 
+# FIXME: update the description
+# NOTE: tha man page is not updated yet!
```

Comment:
What prevents us form updating it at once?

Reply:
@Rezney to be honest, I would need couple of hours to come up how to update the manpage, put the current stuff in the manpage to a new manpage provided by the leapp-repository, .... :) That's why we have the jira ticket about the documentation